### ISSUE TITLE
Game crashing, a few seconds in game

### ISSUE THREAD

**Issue by TastyNut** [ThirteenAG/GTAIV.EFLC.FusionFix#639]
### Game version this happens in

v1.2.0.59

### What area of the game

When launching the game, it loads. After leaving the apartment it only takes a few seconds until the game crashes. 
Example: It crashes quickly if I run towards the food vendor across the Street and punch him.

Info:
**- It's a Vanilla Game, checked file integrity through RGL and it's clean
- Only Mods Used are: FusionFix, XBOX Rain Droplets, Project 2DFX**

In the crashlog, the different types of crashes are:
- Access violation DEP
- Access violation reading location
- Acces violating writing location
(see the .zip)

### What should happen

I guess it shouldn't crash

### CrashDumps and logs

[Save.zip](https://github.com/user-attachments/files/17439653/Save.zip)
[Different Crashdumps.zip](https://github.com/user-attachments/files/17439654/Different.Crashdumps.zip)

### Checklist

- [X] Test in the [latest git build](https://ci.appveyor.com/api/projects/ThirteenAG/gtaiv-eflc-fusionfix/artifacts/GTAIV.EFLC.FusionFix.zip) in case it's already fixed.
- [X] [Search for other reports](https://github.com/ThirteenAG/GTAIV.EFLC.FusionFix/search?q=my+issue&type=issues) of the same issue.
- [X] Include logs or screenshots of issue.

**Comment by ThirteenAG** [comment]
If you open your logs, you will see all crashes happen inside your graphics driver.

**Comment by TastyNut** [comment]
Yeah, but without fusion fix, the game runs perfectly fine, no crashes in 2 hours so far. As soon as i run the game again with fusionfix it crashes quickly again. 

So why is fusionfix incompatible with my system?

**Comment by ThirteenAG** [comment]
No idea.

**Comment by TastyNut** [comment]
In FusionFix.cfg file, having 
`[MAIN]
Windowed = 0` 
causes the crashing.
With it set to **1** I didn't experience any crashes during 20 minutes of rampaging (my form of testing).

**Comment by TastyNut** [comment]
Nevermind, still getting the same crashes, only not as quickly.

**Comment by TastyNut** [comment]
![explorer_7C14CWgRET](https://github.com/user-attachments/assets/2a7cee2a-7e49-4743-82cd-46cf1a545577)

**Comment by Tomasak** [comment]
Did you try removing xbox droplets and p2dfx?

**Comment by TastyNut** [comment]
Yeah, fusionfix alone is causing it, no other mod

**Comment by TastyNut** [comment]
tested again with the new release, still happening.

**Comment by TastyNut** [comment]
btw: I'm using Driver Version 391.95 for NVIDIA Cards on my GTX 570 (latest available).

**Comment by IWasForcedToGetThis** [comment]
Closing this issue as it is pretty old. If you still have this problem then open a new issue.
